src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -991,13 +991,14 @@ class WebappSSLCertTest(ScenarioTest):
     def test_webapp_ssl(self, resource_group, resource_group_location):
         plan = self.create_random_name(prefix='ssl-test-plan', length=24)
         webapp_name = self.create_random_name(prefix='web-ssl-test', length=20)
+        slot_name = self.create_random_name(prefix='slot-ssl-test', length=20)
         # Cert Generated using
         # https://docs.microsoft.com/azure/app-service-web/web-sites-configure-ssl-certificate#bkmk_ssopenssl
         pfx_file = os.path.join(TEST_DIR, 'server.pfx')
         cert_password = 'test'
         cert_thumbprint = '9E9735C45C792B03B3FFCCA614852B32EE71AD6B'
         # we configure tags here in a hope to capture a repro for https://github.com/Azure/azure-cli/issues/6929
-        self.cmd('appservice plan create -g {} -n {} --sku B1 --tags plan=plan1'.format(resource_group, plan))
+        self.cmd('appservice plan create -g {} -n {} --sku S1 --tags plan=plan1'.format(resource_group, plan))
```

Comment:
Because you change the sku value in test,  out of curiosity, is there a requirement for sku to support slot in webapp?

Reply:
Yes :)